tools/doc/json2.js

```diff
@@ -0,0 +1,491 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
Is this a brand new file or is it pulled from another source? brand new files do not require the copyright header to be added.

Reply:
Ultimately, it will be renamed to `tools/doc/json2.js`, and contains significant content from the original source.

Currently separate so that side by side comparisons of the output can be produced, via jsondiff.js, which will be removed when this effort is complete.

Reply:
Ok, thanks for clarification :-)